.circleci/config.yml

```diff
@@ -0,0 +1,73 @@
+version: 2
+
+jobs:
+  build:
+    docker:
+      - image: circleci/node:latest
+    working_directory: ~/fast-dna
+    steps:
+      - checkout
+      - run:
+          name: Install the latest NPM version and patches
+          command: |
+            sudo npm i npm@latest -g
+            npm update rc --depth 4
+      - run:
+          name: Install the latest Lerna version
+          command: |
+            sudo npm i lerna@latest -g
+      - restore_cache:
+          keys:
+          - v1-dependency-cache-{{ checksum "package.json" }}
+      - run:
+          name: Install all dependency packages
+          command: |
+            npm i
+      - run:
+          name: Execute Lerna bootstrap
+          command: |
+            lerna bootstrap
+      - save_cache:
+          key: v1-dependency-cache-{{ checksum "package.json" }}
+          paths:
+            - ./node_modules   
+      - run: 
+          name: Test
+          command: |
+            lerna run test --stream
+
+  "test-coverage":
+    environment:
+      CC_TEST_REPORTER_ID=08a773cb4ea5811add5a45e12873e5cd2634c005568705cc37abfd5217617a32
```

Comment:
should this be a token?

Reply:
It's not required and almost all examples manage it this way.